cartography/intel/gcp/databases.py

```diff
@@ -0,0 +1,864 @@
+    )
+
+@timeit
+def load_sql_users(session: neo4j.Session, data_list: List[Dict[str, Optional[str]]],project_id: str, update_tag: int) -> None:
+    session.write_transaction(_load_sql_users_tx, data_list,project_id, update_tag)
+
+@timeit
+def _load_sql_users_tx(tx: neo4j.Transaction,sql_users: List[Dict],project_id: str,gcp_update_tag: int) -> None:
+    """
+        :type neo4j_transaction: Neo4j transaction object
+        :param neo4j transaction: The Neo4j transaction object
+
+        :type users_resp: List
+        :param users_resp: A list of SQL Users
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :type gcp_update_tag: timestamp
+        :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    """
+    ingest_sql_users = """
+    UNWIND {sql_users} as user
+    MERGE(u:GCPSQLUser{id:{user.id}})
+    ON CREATE SET
+        u.firstseen = timestamp()
+    SET
+        u.name = user.name,
+        u.host = user.host,
+        u.instance = user.instance,
+        u.project = user.project,
+        u.type = user.type
+    WITH user, u
+    MATCH (owner:GCPProject{id:{ProjectId}})-[r1:Resource]->(i:GCPSQLInstance{id:{instance.id}})
+    MERGE (owner)-[r1:RESOURCE]->(i)-[r2:uses]<-(u)
+    ON CREATE SET
+        r1.firstseen = timestamp,
+        r2.firstseen = timestamp,
+        r1.lastupdated = {gcp_update_tag},
+        r2.lastupdated = {gcp_update_tag}
+    """
+    tx.run(
+        ingest_sql_users,
+        sql_users = sql_users,
+        ProjectId=project_id,
+        gcp_update_tag=gcp_update_tag,
+    )
+    
+@timeit
+def load_bigtable_instances(session: neo4j.Session, data_list: List[Dict[str, Optional[str]]], project_id: str, update_tag: int) -> None:
+    session.write_transaction(_load_bigtable_instances_tx, data_list,project_id, update_tag)
+    
+@timeit
+def _load_bigtable_instances_tx(tx: neo4j.Transaction,bigtable_instances: List[Dict],project_id: str,gcp_update_tag: int) -> None:
+    """
+        :type neo4j_transaction: Neo4j transaction object
+        :param neo4j transaction: The Neo4j transaction object
+
+        :type instances_resp: List
+        :param instances_resp: A list of Bigtable Instances
+
+        :type project_id: str
+        :param project_id: Current Google Project Id
+
+        :type gcp_update_tag: timestamp
+        :param gcp_update_tag: The timestamp value to set our new Neo4j nodes with
+    """
+    ingest_bigtable_instances = """
+    UNWIND {bigtable_instances} as instance
+    MERGE (i:GCPBigtableInstance{id:{instance.id}})
+    ON CREATE SET
+        i.firstseen = timestamp()
+    SET
+        i.name = instance.name,
+        i.displayName = instance.displayName,
+        i.state = instance.state,
+        i.type = instance.type,
+        i.createTime = instance.createTime
+    WITH instance, i
+    MATCH (owner:GCPProject{id:{ProjectId}})-[r1:Resource]->(i:GCPBigtableInstance{id:{instance.id}})
```

Comment:
We do not need to match project->resource->BigTableInstance. We can simply match the project and use `i` from MERGE statement in Line 449 to build relationship between project and bigtableinstance